modules/swagger-codegen/src/main/resources/typescript-fetch/index.mustache

```diff
@@ -0,0 +1,5 @@
+// tslint:disable
+{{>licenseInfo}}
+
+export * from "./api";
+export * from "./configuration";
```

Comment:
Yes, this all looks cool. The point of my comment is that there's missing newline https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline.